### DISCUSSION TITLE
StateText without MQTT

### DISCUSSION THREAD

**Discussion by Papablaubaer** [arendst/Tasmota#21782] [Q&A]
I would like to change the StateText in the responses for my HTTP requests. I didn't use MQTT. Unfortunately, the "StateText" command is inactive when MQTT is disabled.

**Comment by sfromis** [comment]
Well, you could temporarily enable MQTT, and make your changes. And do remember that you are getting a less functional setup when not using MQTT, which is what Tasmota is designed for.

**Comment by Papablaubaer** [comment]
Yes, I know that it works like that. I'm trying .autoconf >init.bat.
The solution:
so3 1
stateText1 0
stateText2 1
so3 0
also works. Nevertheless, I think that StateText doesn't really have anything to do with MQTT and should perhaps always be active.

But thank you very much for your help, that doesn't dampen my enthusiasm for the Tasmota project.

**Comment by arendst** [comment]
I'll move StateText from MQTT to "normal" operations now.

**Reply by arendst** [reply]
Done in latest dev.
